codegenerator/cli/templates/static/codegen/src/eventFetching/Source.res

```diff
@@ -41,7 +43,34 @@ type t = {
   ) => promise<result<blockRangeFetchResponse, ErrorHandling.t>>,
 }
 
-let waitForNewBlock = (source, ~currentBlockHeight, ~logger) => {
+let getHeightWithRetry = async (~source, ~logger) => {
+  //Amount the retry interval is multiplied between each retry
+  let backOffMultiplicative = 2
+  //Interval after which to retry request (multiplied by backOffMultiplicative between each retry)
+  let retryIntervalMillis = ref(500)
```

Comment:
Should these be env configurable maybe?

Reply:
I'm going to change the part of the code in the next PR for the fallback feature. I copy-pasted it for now.